### PR TITLE
feat(swagger): transfer changes for draft v3

### PR DESCRIPTION
**Pull request by lewisdaly**
It's my first time contributing to api-snippets! Let me know if I've made any tragic mistakes, or not picked up on the excellent naming conventions.

### CODE REVIEW

File: thirdparty/openapi3/paths/thirdpartyRequests_authorizations.yaml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+# - $ref: ../components/parameters/FSPIOP-Destination.yaml
+# - $ref: ../components/parameters/FSPIOP-Encryption.yaml
+# - $ref: ../components/parameters/FSPIOP-Signature.yaml
+# - $ref: ../components/parameters/FSPIOP-URI.yaml
+# - $ref: ../components/parameters/FSPIOP-HTTP-Method.yaml
+post:
+  description:  >
+    The HTTP request `POST /thirdpartyRequests/authorizations` is used to request the Payer to enter the
+    applicable credentials in the PISP system.
+  operationId: AuthorizationsPostRequest
```
Comment (kleyow) [inline thirdparty/openapi3/paths/thirdpartyRequests_authorizations.yaml:18]: We're going with {HTTPVerb} {Resource} By {Param} {Param} {Param}...etc naming scheme for `operationId` for now as our rough in progress standard.

i.e PostThirdpartyRequestsAuthorizations

Comment (kleyow) [reply thirdparty/openapi3/paths/thirdpartyRequests_authorizations.yaml:18]: Here and throughout

**Comment by kleyow** [comment]
Other than the operation id's make sure to npm run build and commit the generated files as well.

File: thirdparty/openapi3/paths/thirdpartyRequests_authorizations.yaml (hunk 1 above)
Comment (lewisdaly) [reply thirdparty/openapi3/paths/thirdpartyRequests_authorizations.yaml:18]: Nice. Thanks @kleyow

**Review by kleyow — APPROVED** [review]

**Pull request merged**